csharp/msbuild/ice.common.props

```diff
@@ -25,6 +25,9 @@
     <PublishRepositoryUrl>true</PublishRepositoryUrl>
     <RepositoryUrl>https://github.com/zeroc-ice/ice</RepositoryUrl>
     <ProduceReferenceAssembly>false</ProduceReferenceAssembly>
+
+    <!-- Avoid warnings about end-of-line target framework -->
```

Comment:
I initially added this to avoid warnings when targeting .NET 6 with the test suite. I think is best to target .NET 8/.NET 10.

So we can revert this.